.circleci/config.yml

```diff
@@ -54,24 +54,24 @@ jobs:
 
       # Build react-native-desktop in Debug (RCT_DEV enabled)
       - run: |
-          mkdir build_debug
-          pushd build_debug
-          cmake -DCMAKE_BUILD_TYPE=Debug .. && make -j4
+          mkdir build_debug && \
+          pushd build_debug && \
+          cmake -DCMAKE_BUILD_TYPE=Debug .. && make -j4 && \
```

Comment:
Not sure what this is supposed to change...

Reply:
@madhavarshney without making it as one command the whole `sh` CircleCI command doesn't fail if some inner command exits with error code. Because of that all previous ci runs we had compilation with CMake failed, but it was displayed as passed.

Reply:
Doesn't seem to be true: https://circleci.com/gh/status-im/react-native-desktop/566.  I think cmake doesn't exit with a non-zero exit code on configuring errors.

Reply:
Indeed, cmake does according to following logs:
https://circleci.com/gh/MaxRis/react-native-desktop/335
https://circleci.com/gh/MaxRis/react-native-desktop/336
Please compare the output for cmake commands between 336 and 335.

Regarding https://circleci.com/gh/status-im/react-native-desktop/566 it sounds like when you do `exit 1` script executor might exits explicitly as well. But normally it probably returns last command exit code (haven't verified that)

Reply:
> Indeed, cmake does according to following logs:
https://circleci.com/gh/MaxRis/react-native-desktop/335
https://circleci.com/gh/MaxRis/react-native-desktop/336
Please compare the output for cmake commands between 336 and 335.

My bad, that seems to be true.

> But normally it probably returns last command exit code (haven't verified that)

Not according to [the CircleCI docs](https://circleci.com/docs/2.0/configuration-reference/#default-shell-options).  Maybe I misunderstood the docs.

Reply:
Just found [this forum thread](https://discuss.circleci.com/t/tests-fail-but-build-continues-and-finishes-successfully/10450).  Seems like you're right, although the official docs somewhat contradict that thread.

Reply:
I think `set -e` in each step would probably be neater.